tests/test_client/tests.py

```diff
@@ -41,50 +43,107 @@ def setUpTestData(cls):
         cls.u2 = User.objects.create_user(username='inactive', password='password', is_active=False)
 
     def test_get_view(self):
-        "GET a view"
+        """GET a view"""
         # The data is ignored, but let's check it doesn't crash the system
         # anyway.
         data = {'var': '\xf2'}
         response = self.client.get('/get_view/', data)
 
-        # Check some response details
         self.assertContains(response, 'This is a test')
         self.assertEqual(response.context['var'], '\xf2')
         self.assertEqual(response.templates[0].name, 'GET Template')
 
     def test_get_post_view(self):
-        "GET a view that normally expects POSTs"
+        """GET a view that normally expects POSTs"""
```

Comment:
This and the few following cleanups are strictly unrelated to this patch. 

Presumably, they're here because of the feedback for similar items on #9636. There are lots of inconsistencies left in this file, around both docstrings and formatting of simple (e.g.) `post_data` dicts. 

Despite being strictly unrelated, I'm inclined to leave these few changes as they are here, and include them.

Reply:
Yep I'm starting to think I'll make a ticket to clean up this file.

Mind if I revert these back to how they were for that ticket instead?

Reply:
@NDevox I'd be very happy for you to do that. 🙂

(If it's just a single file cleanup, you can just make a PR, without the Trac ticket)